pcli-next/src/command/balance.rs

```diff
@@ -0,0 +1,109 @@
+use anyhow::Result;
+use comfy_table::{presets, Table};
+use penumbra_crypto::{keys::DiversifierIndex, FullViewingKey, Value};
+use penumbra_view::ViewClient;
+use structopt::StructOpt;
+
+// TODO
```

Comment:
stray TODO?

Reply:
Yeah, looks like it. I think I might have been about to suggest that we move `balance` into a subcommand of a new `view` command.